sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/DocumentInputTest.java

```diff
@@ -42,29 +42,29 @@ protected static void afterTest() {
 
     // Detect language
     /**
-     * Verifies that a null pointer exception is thrown when null document is given for
+     * Verifies that a {@link NullPointerException} is thrown when null document is given for
      * {@link TextAnalyticsClient#detectLanguage(String)}
      */
     @Test
-    public void detectLanguagesNullInput() {
+    public void detectLanguageNullInput() {
         Exception exception = assertThrows(NullPointerException.class, () ->
             client.detectLanguage(null));
         assertTrue(INVALID_DOCUMENT_NPE_MESSAGE.equals(exception.getMessage()));
     }
 
     /**
-     * Verifies that a null pointer exception is thrown when null document is given for
+     * Verifies that a {@link NullPointerException} is thrown when null document is given for
```

Comment:
Can you add a test for warnings? We're only able to add it for extract key phrases, but this is the python test: https://github.com/Azure/azure-sdk-for-python/blob/9da31763f9809305c4e04ac41b7a6a995dcdeb9e/sdk/textanalytics/azure-ai-textanalytics/tests/test_extract_key_phrases.py#L411